Jellyfin.Plugins.PushOverNotifications/Notifier.cs

```diff
@@ -2,23 +2,23 @@
 using MediaBrowser.Common.Net;
 using MediaBrowser.Controller.Entities;
 using MediaBrowser.Controller.Notifications;
-using MediaBrowser.Model.Logging;
+using Microsoft.Extensions.Logging;
 using MediaBrowser.Plugins.PushOverNotifications.Configuration;
 using System;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace MediaBrowser.Plugins.PushOverNotifications
+namespace Jellyfin.Plugins.PushOverNotifications
 {
     public class Notifier : INotificationService
     {
         private readonly ILogger _logger;
         private readonly IHttpClient _httpClient;
 
-        public Notifier(ILogManager logManager, IHttpClient httpClient)
+        public Notifier(ILoggerFactory logManager, IHttpClient httpClient)
         {
-            _logger = logManager.GetLogger(GetType().Name);
+            _logger = logManager.CreateLogger(GetType().Name);
```

Comment:
Here I'd rename the variable to loggerFactory. The base class is just called Notifier so maybe a manual name that identifies the plugin is better here

Reply:
So change _logger to loggerFactory? Sorry I don't really know what you mean XD